src/object/schema_template.c

```diff
@@ -4334,8 +4334,8 @@ smt_change_attribute (SM_TEMPLATE * template_, const char *name, const char *new
       assert (name_space == ID_ATTRIBUTE);
     }
 
-  att->type = new_domain->type;
-  att->domain = new_domain;
+  // att->type = new_domain->type;
```

Comment:
remove the code if the codes not needed

Reply:
Thank you. This part was commented out in the middle of the implementation. It was restored to its original state in the last commit. The smt_change_attribute function is not called while the do_alter_change_col_comment function is running.